packages/web-components/src/components/card-link/__stories__/README.stories.mdx

```diff
@@ -33,7 +33,8 @@ import '@carbon/ibmdotcom-web-components/es/components/card-link/index.js';
 
 ```html
 <dds-card-link href="https://example.com">
-  Copy text
+  <dds-card-heading>Heading text</dds-card-heading>
```

Comment:
Since we're now referencing `<dds-card-heading>`, let's ensure that the [index.ts](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/card-link/index.ts) of this component is importing from `import '../card/card-heading;`, so all of the dependent components are imported in the build process.